addons/sale/static/src/scss/product_configurator.scss

```diff
@@ -133,7 +157,7 @@ label.css_attribute_color.css_not_available {
     }
 
     .td-qty {
-        width: 130px;
```

Comment:
can you double check that you don"t break ecommerce to fix product configurator please ! 

Thanks